resources/assets/js/components/AssessmentPlan/RatingGuideClipboard.test.ts

```diff
@@ -19,6 +19,7 @@ import {
   criteriaType,
 } from "../../models/localizedConstants";
 import { clipboardData, ClipboardTableRowProps } from "./RatingGuideClipboard";
+import { localizeField } from "../../helpers/localize";
```

Comment:
Looks like all the methods in this file need to be changed to `localizeFieldNonNull()`